ons_alpha/bulletins/models.py

```diff
@@ -61,22 +59,43 @@ class BulletinPage(BasePage):
         ]
     )
 
-    def get_context(self, request, *args, **kwargs):
-        context = super().get_context(request, *args, **kwargs)
-
-        context["toc"] = self.toc
-
-        return context
+    @property
+    def is_latest(self):
+        latest_id = (
+            BulletinPage.objects.sibling_of(self).live().order_by("-release_date").values_list("id", flat=True).first()
+        )
+        return self.pk == latest_id
 
     @cached_property
     def toc(self):
         items = [{"url": "#summary", "text": "Summary"}]
-
         for block in self.body:  # pylint: disable=not-an-iterable
             if hasattr(block.block, "to_table_of_contents_items"):
                 items += block.block.to_table_of_contents_items(block.value)
-
         if self.contact_details_id:
             items += [{"url": "#contact-details", "text": "Contact details"}]
-
         return items
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context["toc"] = self.toc
+        return context
+
+
+class BulletinSeriesPage(RoutablePageMixin, Page):
+    template = "templates/pages/bulletins/series_page.html"
+    parent_page_types = ["home.HomePage"]
+    subpage_types = ["BulletinPage"]
+
+    @path("latest/")
+    def latest_bulletin(self, request):
+        latest = BulletinPage.objects.live().child_of(self).order_by("-release_date").first()
+        return render(request, "templates/pages/bulletins/bulletin_page.html", {"page": latest})
+
+    @path("previous-releases/")
+    def previous_releases(self, request):
+        previous = BulletinPage.objects.live().child_of(self).order_by("-release_date")
+        return render(request, "templates/pages/bulletins/previous_releases.html", {"bulletins": previous})
+
+    def serve(self, request, *args, **kwargs):
+        return redirect(self.url + self.reverse_subpage("latest_bulletin"))
```

Comment:
Nit: It's bad practice to use `.url` directly. By using `.get_url(request)`, Wagtail can cache some things on the request, and getting the URL is faster (as it doesn't need to do any DB queries).